source/frontend/src/features/mapSideBar/property/tabs/takes/update/TakesUpdateContainer.test.tsx

```diff
@@ -80,20 +80,24 @@ describe('TakesUpdateContainer component', () => {
 
   it('renders as expected', async () => {
     const { asFragment } = setup({});
-    await waitForEffects();
+    await act(async () => {});
```

Comment:
I believe this is equivalent to `await waitForEffects()` - did you find any differences in behavior in the tests?

Reply:
await waitForEffects() does not appear to work in all cases with the newer version of jest that we are currently on. In all cases where I replaced that function, the test threw warnings before the replacement but stopped after the replacement (I did not touch any tests that were not throwing warnings).

Reply:
agree we could look into modifying the waitForEffects method later